src/Api/Vault/AuthorizationHandlers/Collections/BulkCollectionAuthorizationHandler.cs

```diff
@@ -324,10 +319,8 @@ private async Task<bool> CanDeleteAsync(ICollection<Collection> resources, Curre
         return await _applicationCacheService.GetOrganizationAbilityAsync(organization.Id);
     }
 
-    private async Task<bool> AllowAdminAccessToAllCollectionItems(CurrentContextOrganization? org)
+    private async Task<bool> AllowAdminAccessToAllCollectionItems(CurrentContextOrganization? org = null, OrganizationAbility? orgAbility = null)
```

Comment:
I assume this change is to avoid fetching the `orgAbility` twice. However, the org abilities are stored in an in-memory cache, so retrieval is basically free. The idea of this private method is so that you don't need to know about the org cache, so adding it here as an optional parameter is muddying its purpose.

I recommend reverting changes to this interface, and then:
1. don't worry about hitting the cache twice, or
2. if you prefer, just don't use it in `CanDeleteAsync`, which is the only location where the orgAbility is needed twice in a single method call.

Reply:
Correct. I went with option 1. Thanks @eliykat